src/main/java/com/scalar/cassy/CassyBackupScheduler.java

```diff
@@ -0,0 +1,112 @@
+package com.scalar.cassy;
+
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "cassy-schedule",
+    mixinStandardHelpOptions = true,
+    description = "calls Cassy's backup command",
+    subcommands = {ClusterSnapshot.class, NodeSnapshot.class, NodeIncremental.class})
+public class CassyBackupScheduler implements Runnable {
```

Comment:
For readability reason, I think it will be great to split each command in a separate file.

Reply:
Funnily enough I had it in separate files initially, but after moving most of the logic to the Client class I refactored them into one file xD

I'll split it into different files again.

Reply:
Ah 😄